util/tooltool/tooltool.py

```diff
@@ -833,39 +836,39 @@ def upload(manifest, message, base_urls, auth_file, region):
     # Upload the files, each in a thread.  This allows us to start all of the
     # uploads before any of the URLs expire.
     threads = {}
-    for filename, file in files.iteritems():
-        if 'put_url' in file:
-            log.info("%s: starting upload", filename)
+    for filename, file_ in files.iteritems():
+        if 'put_url' in file_:
+            log.info("%s: starting upload", (filename,))
             thd = threading.Thread(target=_s3_upload,
-                                   args=(filename, file))
+                                   args=(filename, file_))
             thd.daemon = 1
             thd.start()
             threads[filename] = thd
         else:
-            log.info("%s: already exists on server", filename)
+            log.info("%s: already exists on server", (filename,))
 
     # re-join all of those threads as they exit
     success = True
     while threads:
         for filename, thread in threads.items():
             if not thread.is_alive():
                 # _s3_upload has annotated file with result information
-                file = files[filename]
+                file_ = files[filename]
                 thread.join()
-                if file['upload_ok']:
+                if file_['upload_ok']:
                     log.info("%s: uploaded", filename)
                 else:
-                    log.error("%s: failed", filename, exc_info=file['upload_exception'])
+                    log.error("%s: failed", filename, exc_info=file_['upload_exception'])
                     success = False
                 del threads[filename]
 
     # notify the server that the uploads are completed.  If the notification
     # fails, we don't consider that an error (the server will notice
     # eventually)
-    for filename, file in files.iteritems():
-        if 'put_url' in file and file['upload_ok']:
-            log.info("notifying server of upload completion for %s", filename)
-            _notify_upload_complete(base_urls[0], auth_file, file)
+    for filename, file_ in files.iteritems():
+        if 'put_url' in file_ and file_['upload_ok']:
+            log.info("notifying server of upload completion for %s", (filename,))
```

Comment:
[Ditto here](https://github.com/MozillaSecurity/funfuzz/commit/5922971e63f471851a54ad8b40550b12eed1b5f6#diff-9018c66b02761bd8e129f084cfae1539L883).

Would you still like me to fix these to remove the tuples?